README.md

```diff
@@ -28,6 +28,10 @@ Survey
 * [bioRxiv 2022] **Learning inverse folding from millions of predicted structures**. [[Paper]](https://www.biorxiv.org/content/10.1101/2022.04.10.487779v1.abstract)[[Code]](https://github.com/facebookresearch/esm)
 <br><br>
 * [PLOS 2022] **Ig-VAE: Generative modeling of protein structure by direct 3D coordinate generation**. [[Paper]](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1010271)[[Code]](https://github.com/ProteinDesignLab/IgVAE)
+<br><br>
+* [ICML 2022] **LIMO: Latent Inceptionism for Targeted Molecule Generation**. [[Paper]](https://arxiv.org/abs/2206.09010)[[Code]](https://github.com/rose-stl-lab/limo)
+* [ICML 2022] **Generating 3D Molecules for Target Protein Binding**. [[Paper]](https://arxiv.org/abs/2204.09410)[[Code]](https://github.com/divelab/GraphBP)
```

Comment:
I believe we have already added GraphBP to this list, but thank you for pointing it out.